### PR TITLE
Add SUITE_CONTEXT property, 'revoked' export tests.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
See also corresonding PR https://github.com/digitalbazaar/crypto-ld/pull/58

### CODE REVIEW

File: test/X25519KeyAgreementKey2020.spec.js [hunk 1]
```diff
@@ -117,12 +119,17 @@ describe('X25519KeyAgreementKey2020', () => {
     });
 
     it('should export both public and private key', async () => {
-      const key = await X25519KeyAgreementKey2020.generate();
+      const key = await X25519KeyAgreementKey2020.generate({
+        controller: 'did:example:1234'
+      });
+      key.revoked = '2019-10-12T07:20:50.52Z';
```
Comment (mattcollier) [inline test/X25519KeyAgreementKey2020.spec.js]: Please use the new one liner for RFC3339

https://github.com/digitalbazaar/http-signature-zcap-verify/blob/f3a100c3cfcff4cfde24614b61ab7c03b6f6efb8/tests/10-verify.spec.js#L178-L179

**Review by mattcollier — CHANGES_REQUESTED** [review]

File: test/X25519KeyAgreementKey2020.spec.js (hunk 1 above)
Comment (dmitrizagidulin) [reply test/X25519KeyAgreementKey2020.spec.js]: Changed.

File: test/X25519KeyAgreementKey2020.spec.js [hunk 2]
```diff
@@ -117,12 +119,19 @@ describe('X25519KeyAgreementKey2020', () => {
...
+      const pastDate = new Date(2020, 11, 17).toISOString()
+        .replace(/\.[0-9]{3}/, '');
+      key.revoked = pastDate;
 
       const exported = key.export({publicKey: true, privateKey: true});
-      expect(exported).to.have.keys([
-        'id', 'controller', 'type', 'publicKeyMultibase', 'privateKeyMultibase'
-      ]);
+      expect(exported).to.have.keys(['id', 'type', 'controller',
+        'publicKeyMultibase', 'privateKeyMultibase', 'revoked']);
```
Comment (mattcollier) [inline test/X25519KeyAgreementKey2020.spec.js]: sorry, missed this before: style, trying to eyeball 80 char limit.
```suggestion
      expect(exported).to.have.keys([
        'id', 'type', 'controller', 'publicKeyMultibase',
        'privateKeyMultibase', 'revoked'
      ]);
```

**Review by mattcollier — APPROVED** [review]
One more style nit.

**Pull request merged**